sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -230,7 +230,7 @@ private[joins] class UnsafeHashedRelation(
       new UnifiedMemoryManager(
         new SparkConf().set(MEMORY_OFFHEAP_ENABLED.key, "false"),
         Long.MaxValue,
-        Long.MaxValue,
+        Long.MaxValue / 2,
```

Comment:
note: This actually doesn't matter. It's a dummy memory manager which is only used to create the `BytesToBytesMap`. This memory manager will not consume any storage memory, which means the execution memory can grow until it takes all the memory, which is `Long.MaxValue`.

Anyway, it's still better to make sure the values are not nonsense.